app/src/main/res/layout/fragment_settings.xml

```diff
@@ -65,6 +65,58 @@
 
         <View style="@style/Border" />
 
+        <RelativeLayout android:id="@+id/notification_setting_container"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:background="@drawable/clickable"
+            android:padding="@dimen/spacing">
```

Comment:
I guess it is better that user can click this layout to switch change. How do you think?